src/components/icons/pages/icons.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import DocsBlock, {contrastBlockCssClass} from '../../../docs/DocsBlock';
```

Comment:
DocsBlock it's internal component for build docs, we don't export it like other components